pulp_container/app/viewsets.py

```diff
@@ -359,6 +359,46 @@ def copy_manifests(self, request, pk):
         )
         return OperationPostponedResponse(result, request)
 
+    @swagger_auto_schema(
+        operation_description="Trigger an asynchronous task to build an OCI image from a "
+                              "Dockerfile. A new repository version is created with the new image "
+                              "and tag. This API is tech preview in Pulp Container 1.1. Backwards "
+                              "compatibility when upgrading is not guaranteed.",
+        operation_summary="Build a Dockerfile",
+        responses={202: AsyncOperationResponseSerializer},
+        request_body=serializers.OCIBuildImageSerializer,
+    )
+    @action(detail=True, methods=['post'], serializer_class=serializers.TagImageSerializer)
+    def build_dockerfile(self, request, pk):
```

Comment:
what do you think about renaming to 'build_image'

Reply:
i like it. i'll rename.